framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -154,7 +154,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
         // remove the old binding
         if (old != null && !old.isDisposed() && ids[property] != null)
         {
-          old.removeBinding(ids[property]);
+          this.removeBinding(ids[property]);
```

Comment:
The original binds from the Command to the Button, i.e. I'm asking myself what the reason for this is, actually. That breaks the enabled state for example. So I changed the direction in the commit. The Command still works and the button still works.

I changed the binding and the unbinding direction. this->value and this->old.